src/cache/cache.rs

```diff
@@ -94,5 +119,184 @@ impl Cache {
     pub fn get_default_port(&self) -> u16 {
         self.port
     }
+
+    pub fn clear_expired(&self) {
+        let mut store = self.store.lock().unwrap();
     
+        for cluster_store in store.values_mut() {
+            cluster_store.retain(|_, (_, expiration_time, _)| {
+                if let Some(exp) = expiration_time {
+                    exp > &mut Instant::now()
+                } else {
+                    true
+                }
+            });
+        }
+    }
+
+    // Eviction strategies
+    fn evict_entries(&self) {
+        let mut store = self.store.lock().unwrap();
+        let mut memory_handler = self.memory_handler.lock().unwrap();
+        let eviction_strategy = EvictionStrategy::from_i32(self.evict_type);
+        match eviction_strategy.unwrap() {
+            EvictionStrategy::VolatileLru => self.evict_volatile_lru(&mut store, &mut memory_handler),
+            EvictionStrategy::AllKeysLru => self.evict_allkeys_lru(&mut store, &mut memory_handler),
+            EvictionStrategy::AllKeysRandom => self.evict_allkeys_random(&mut store, &mut memory_handler),
+            //EvictionStrategy::VolatileRandom => self.evict_volatile_random(&mut store, &mut memory_handler),
```

Comment:
why this is commented?
if its unnecessary remove it.

Reply:
no I want fix that later 
I need this scope of codes

Reply:
its better to remove unused code.